src/app/features/management/user-management-view/user-management-view.component.ts

```diff
@@ -44,10 +45,11 @@ export class UserManagementView {
   }
 
   editUser(userId: string): void {
```

Comment:
probably rename to something like 'viewUser' or 'navigateToUser' as intention is not always editing.